components/Send/AmountInput.tsx

```diff
@@ -45,31 +39,74 @@ export const AmountInput = ({ onChange, value }: AmountInputProps) => {
     }
   };
 
+  const handleTokenSelect = (token: Token) => {
+    setSelectedToken(token);
+    setIsPickerOpen(false);
+    onTokenChange && onTokenChange(token);
+  };
+
   return (
-    <View className="mb-4 h-40 rounded-xl bg-content p-4">
```

Comment:
Why are we removing the fixed height? The Recipient Input also has this

Reply:
Recippient input has it so the component doesn't change height due to the profile picture appearing when an ENS was entered, not needed here as nothing changes size.

Reply:
But the two components do not have the same size due to this

Reply:
Do they have to be the same size? If yes then i will re-implement the size.